src/extension/background-script/actions/setup/getPassword.ts

```diff
@@ -0,0 +1,10 @@
+import { MessageGetPassword } from "~/types";
+
+import state from "../../state";
+
+const getPassword = (message: MessageGetPassword) => {
+  const passwordExists = state.getState().password ? true : false;
```

Comment:
I think we have the status action for this: https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/actions/setup/status.js